src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -249,6 +250,11 @@ All parameters are case-insensitive.</value>
 
 -WindowStyle | -w
     Sets the window style to Normal, Minimized, Maximized or Hidden.
+
+-WorkingDirectory | -wd
+    Sets the working directory at the start of PowerShell given a valid PowerShell directory path.
```

Comment:
Probably should mention that this "valid PowerShell directory path" is used as a LiteralPath.

Reply:
Will add

Reply:
Should be true for all well written providers

Reply:
Make sense add this to the help or online help?